projects/observability/src/pages/apis/api-detail/traces/api-trace-list.dashboard.ts

```diff
@@ -58,6 +58,19 @@ export const apiTraceListDashboard = {
           type: 'api-trace-navigation-handler'
         }
       },
+      {
+        type: 'table-widget-column',
+        title: 'Errors',
+        width: '180px',
```

Comment:
This is just a number right? 180px feels a bit excessive

Reply:
Please refer to: https://github.com/hypertrace/hypertrace-ui/pull/786